pkg/controller/migplan/storage.go

```diff
@@ -130,10 +133,31 @@ type PlanStorage struct {
 	storage       *migapi.MigStorage
 }
 
+// Get labels (correlation labels+migrationcontroller label)
+func (r PlanStorage) getLabels() map[string]string {
+	labels := r.plan.GetCorrelationLabels()
+	migrationController := &unstructured.Unstructured{}
+	gvk := schema.GroupVersionKind{
+		Kind:    "migrationcontroller",
+		Group:   "migration.openshift.io",
+		Version: "v1alpha1",
+	}
+	migrationController.SetGroupVersionKind(gvk)
+	migrationControllerName := types.NamespacedName{
+		Namespace: migapi.VeleroNamespace,
+		Name:      "migration-controller",
```

Comment:
If the user changes the name of this CR this feature won't work correct?

Reply:
I had assumed that the CR name was controlled by the operator. Is that not the case?

Reply:
I could pull the list and just take the UID of the first one, but it might be better to just not annotate if the operator-created one isn't there.

Reply:
The user creates this from OLM generally, which defaults to this name but it's not the operator that does it.

Grabbing the first one in the list could work since there should only be one... but I'm not sure on the best way to proceed. 

Maybe we get the list, if there is more than one we just don't set it?

Reply:
I've updated this to just pull the UID from the first one found.